### DISCUSSION TITLE
Update gems, but only the new release is "a bit" old

### DISCUSSION THREAD

**Discussion by will** [ruby/rubygems#5404] [Ideas]
I like to keep my project up to date as much as possible, but I think there'd be some benefit to automatically letting the release "bake" for a while before pulling them in. Might help to avoid some newly introduced bugs, and might help a bit on the problem of malicious releases. Potentially gives you some time to at least hear about a problem at the very least.

It's too tedious to do that manually though. Would it be interesting if there was an option to update all your gems, but exclude releases if they're too new by some measure?

**Comment by will** [comment]
:(

**Reply by duckinator** [reply]
Sorry about that, not sure why this one got closed.

**Comment by felix-as** [comment]
Feature is called "cooldown period" and discussed here: https://github.com/ruby/rubygems/discussions/9113 . I propose to close this issue in favor of the newer and more discussed one. @duckinator

**Reply by duckinator** [reply]
I'm not on the RubyGems team anymore, but it looks like they did what you proposed anyway. :slightly_smiling_face:
